guidelines/enhancement_template.md

```diff
@@ -19,6 +19,21 @@ superseded-by:
   - "/enhancements/our-past-effort.md"
 ---
 
+Start by filling out this header template with metadata for this enhancement.
+
+* `reviewers`: This can be anyone that has an interest in this work.
+
+* `approvers`: All enhancements must be approved, but the appropriate people to
+  approve a given enhancement depends on its scope.  If an enhancement is
+  limited in scope to a given team or component, then a peer or lead on that
+  team is an appropriate approver.  If an enhancement captures something more
```

Comment:
```suggestion
  team or pillar is an appropriate approver.  If an enhancement captures something more
```

Reply:
done!

Reply:
I expect everyone will understand "team", but "pillar" is a Red-Hat-ism.  Do we have docs we can link to provide some context?